examples/ai_society/babyagi_playing.py

```diff
@@ -0,0 +1,58 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+
+from colorama import Fore
+
+from camel.societies import BabyAGI
+from camel.utils import print_text_animated
+
+
+def main(model_type=None) -> None:
+    task_prompt = "Develop a trading bot for the stock market"
+    babyagi_session = BabyAGI(
+        assistant_role_name="Python Programmer",
+        assistant_agent_kwargs=dict(model=model_type),
```

Comment:
We can also pass `message_window_size` via `*_agent_kwargs` like:
```suggestion
        assistant_agent_kwargs=dict(model=model_type, message_window_size=5),
```

Reply:
I would vote we adopt the original way, passing message_window_size as a separate parameter.
1, message_window_size is important to achieve good performance and worth a parameter.
2,  message_window_size is also used in task_creation_agent and task_prioritization_agent initialization, while assistant_agent_kwargs only used in assistant_agent.

Reply:
I am good with the code by @hychen-naza